staging/src/k8s.io/client-go/scale/interfaces.go

```diff
@@ -34,11 +36,11 @@ type ScalesGetter interface {
 // the scale subresource.
 type ScaleInterface interface {
 	// Get fetches the scale of the given scalable resource.
-	Get(resource schema.GroupResource, name string) (*autoscalingapi.Scale, error)
+	Get(ctx context.Context, resource schema.GroupResource, name string, opts v1.GetOptions) (*autoscalingapi.Scale, error)
```

Comment:
Are the options ever going to be used here or are they always going to be empty?

Reply:
Ah likely empty since GetOptions aren’t currently useful. I included them only for consistency.

Reply:
Consistency makes sense given that they may be useful in the future.